R/cache.R

```diff
@@ -32,7 +32,7 @@ load_cache <- function(file, path = NULL) {
 
   file <- get_cache_file_path(file, path)
   if (file.exists(file)) {
-    load(file = file, envir = env)
+    tryCatch(load(file = file, envir = env), error = NULL)
```

Comment:
I think we don't need to stop on error here, but a warning would still be warranted.

Reply:
Sure. Turning the error into a warning looks reasonable.